README.md

```diff
@@ -14,13 +14,34 @@ Includes SingularityNET platform contracts, migrations, tests
 ### MultiPartyEscrow
 * Contract which support payment channels in SingularityNet.
 
-## Interfaces
+## Interfaces (npm version 0.3.2)
+
+| Name      | InterfaceID ([ERC-165](https://eips.ethereum.org/EIPS/eip-165)) | Source Code                              |
+|-----------|-----------------------------------------------------------------|------------------------------------------|
+| IRegistry | 0x1d466fef                                                      | [IRegistry.sol](contracts/IRegistry.sol) |
+
+## Deployed Contracts (npm version 0.3.2)
+
+* Registry (Mainnet): [0x2d367d5b442135b29a4705871e4c7d34382cb442](https://etherscan.io/address/0x2d367d5b442135b29a4705871e4c7d34382cb442)
+* Multi Party Escrow (Mainnet): [0xd519dfc46263aebaa4fa43c7666c091016c344ce](https://etherscan.io/address/0xd519dfc46263aebaa4fa43c7666c091016c344ce)
+* AGI Token (Mainnet) :  [0x8eb24319393716668d768dcec29356ae9cffe285](https://etherscan.io/address/0x8eb24319393716668d768dcec29356ae9cffe285)
+
+* Registry (Kovan): [0x65d13448b00ed15598de35532e4131b816692986](https://kovan.etherscan.io/address/0x65d13448b00ed15598de35532e4131b816692986)
+* Multi Party Escrow (Kovan): [0x45ae5546b1f7cca0e58bd5146590a215a8d17583](https://kovan.etherscan.io/address/0x45ae5546b1f7cca0e58bd5146590a215a8d17583)
+* AGI Token (Kovan) :  [0x3b226fF6AAd7851d3263e53Cb7688d13A07f6E81](https://kovan.etherscan.io/address/0x3b226fF6AAd7851d3263e53Cb7688d13A07f6E81)
+
+* Multi Party Escrow (Ropsten) :  [0x51ec07787757f9a12cf8368f99145bfb4c5aa70e](https://ropsten.etherscan.io/address/0x51ec07787757f9a12cf8368f99145bfb4c5aa70e)
```

Comment:
The title is mixed up, its should be registry first and then the  MPE

Reply:
Thanks, updated.